script.js

```diff
@@ -1,4 +1,123 @@
+var player1Score = 0;
+var player2Score = 0;
+//total score for player
+var totalPlayer1Score = [];
+var totalPlayer2Score = [];
+var totalSumPlayer1 = 0;
+var totalSumPlayer2 = 0;
+//dices for players
+var player1Dice1 = 0;
+var player1Dice2 = 0;
+var player2Dice1 = 0;
+var player2Dice2 = 0;
+
+var gameMode = `player one roll`;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (gameMode == `player one roll`) {
```

Comment:
I like that you chose an intuitive name for the mode, that literally descibes what is supposed to be done in that mode. Consider saving these as constants with scream case (i.e. MODE_PLAYER_ONE_ROLL)